content/blog/kubernetes-chart-v4/index.md

```diff
@@ -0,0 +1,846 @@
+
+    }
+}
+```
+
+{{% /choosable %}}
+
+{{% choosable language yaml %}}
+
+```yaml
+resources:
+  certman:
+    type: kubernetes:helm.sh/v4:Chart
+    properties:
+      namespace: cert-manager
+      chart: oci://registry-1.docker.io/bitnamicharts/cert-manager
+      version: "1.3.1"
+```
+
+{{% /choosable %}}
+
+{{< /chooser >}}
+
+### OCI Registry Support
+
+You can use container registries with OCI support such as [Docker Hub](https://hub.docker.com/) to store and share
+Helm chart packages. The Chart v4 resource now has full support for OCI, bringing it to parity with
+the Release resource.
+
+To use an authenticated OCI registry, you must first login using `helm registry login` or `docker login`.
+
+Chart v4 also supports the use of [Helm chart repositories](https://helm.sh/docs/topics/chart_repository/),
+and adopts the same `repositoryOpts` API as was introduced in the Release resource.
+
+### Lock File Support
+
+Helm has support for lock files (Chart.lock) to control a chart's dependencies. When deploying a chart from a local directory,
+Pulumi automatically rebuilds the chart's dependencies if a lock file is present. See
+[Helm Dependency Build](https://helm.sh/docs/helm/helm_dependency_build/) for details.
+
+### Better Handling of Chart Values
+
+Chart v4 offers new ways to work with Chart values. It is now possible to use multiple values files and to use
+[Pulumi Assets](/docs/concepts/assets-archives/#assets). Of course you can also use output values from other
+resources as chart values.
+
+It is also possible to set a chart value to the contents of a text file, similarly to using Helm's `--set-file` argument.
+To do that, simply use a Pulumi Asset as a value within the `values` map.
+
+### Better Connectivity
+
+You may now use chart functions that require a connection to the cluster, e.g. to:
+
+- Check the Kubernetes server version with [`.Capabilities.KubeVersion`](https://helm.sh/docs/chart_template_guide/builtin_objects/)
+- Check if an API version or kind is available with [`.Capabilities.APIVersions.Has`](https://helm.sh/docs/chart_template_guide/function_list/#capabilitiesapiversionshas)
+- Use the [`lookup` function](https://helm.sh/docs/chart_template_guide/function_list/#lookup)
+in your templates to get existing objects in your live cluster.
+
+Note that the lookup function is executed in both the preview and the non-preview mode, and keep in mind that
+the expected object may not exist during a preview.
+
+### Improved Resource Ordering
+
+It's now easy to wait for a chart's resources to be installed before installing other resources,
+simply by using the `dependsOn` option. In earlier versions, we relied on a `ready` output property.
+
+The Chart resource automatically detects dependencies between resources in the manifest(s).
+For example, it knows to install namespaces and Custom Resource Definitions (CRDs) first.
+
+Use the `config.kubernetes.io/depends-on` annotation to declare an explicit resource dependency.
```

Comment:
Example?

Reply:
I feel there's no need because the post already has a transformation example that shows how to add annotations.